src/hwc/integration/integration_suite_test.go

```diff
@@ -34,7 +34,7 @@ func init() {
 var _ = SynchronizedBeforeSuite(func() []byte {
 	// Run once
 	if buildpackVersion == "" {
-		packagedBuildpack, err := cutlass.PackageUniquelyVersionedBuildpack(os.Getenv("CF_STACK"), ApiHasStackAssociation())
+		packagedBuildpack, err := cutlass.PackageUniquelyVersionedBuildpack("", ApiHasStackAssociation())
```

Comment:
We had to leave out stack specification for this to pass. Our understanding is this will be resolved once the cross-cutting stack association work is complete but for now, this buildpack will continue to work since hwc-buildpack apps are pushed with explicit stacks.